lib/generators/devise/devise_generator.rb

```diff
@@ -13,7 +13,7 @@ class DeviseGenerator < Rails::Generators::NamedBase
       desc "Generates a model with the given NAME (if one does not exist) with devise " \
            "configuration plus a migration file and devise routes."
 
-      hook_for :orm
+      hook_for :orm, required: true, desc: "ORM to be invoked"
```

Comment:
I don't think this option is required. Doesn't `type: :boolean` fix this issue?

Reply:
Both work. Since it is not required, I updated the commit. You can merge.

Reply:
I tried changing back to the initial proposal here: `required: true`, and it fixes https://github.com/heartcombo/devise/issues/5252. Since `rails` uses the same thing: https://github.com/rails/rails/blob/b1646a48fafd7a2d2c2aad3b3fc77764d7a37bed/railties/lib/rails/generators/rails/application_record/application_record_generator.rb#L6, it sounds like that's the correct thing and the option is indeed required?

Reply:
Sounds like it yeah, can you open a PR? Thanks.

Reply:
Absolutely!